packages/compass-aggregations/src/components/stage-editor/stage-editor.tsx

```diff
@@ -143,7 +143,7 @@ export const StageEditor = ({
         {
           num_stages: num_stages,
           stage_index: index + 1,
-          stage_action: 'stage_content_changed',
+          stage_action: 'stage_content_changed' as const,
```

Comment:
I'm not understanding what these `as const` bits are doing and why they would be necessary.

Reply:
as in move `as const` at the end? They are there so `stage_action` is not considered a string

Reply:
ah wait, you are right this should not be needed here though. I'll take another look and remove them where is redundant